src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,46 @@
+package duke.parser;
+
+import duke.command.*;
```

Comment:
Perhaps consider importing specific packages instead of using the wildcard import.

Reply:
Agree